index.md

```diff
@@ -11,42 +11,42 @@ Feel free to explore, and if anything catches your attention or raises questions
 
 ## Project Kickoff
 
-- [Project Charter](/project_kickoff/project_charter.md)
-- [Sprint Ceremonies](/project_kickoff/ceremonies.md)
-- [Definition of Ready](/project_kickoff/definition-of-ready.md)
-- [Definition of Done](/project_kickoff/dod.md)
-- [Team Norms](/project_kickoff/team_norms.md)
-- [Meeting Etiquette](/project_kickoff/meeting-etiquette.md)
-- [Securing Your Workspace](/project_kickoff/securing_your_workspace.md)
+- [Project Charter](project_kickoff/project_charter.md)
+- [Sprint Ceremonies](project_kickoff/ceremonies.md)
+- [Definition of Ready](project_kickoff/definition-of-ready.md)
+- [Definition of Done](project_kickoff/dod.md)
+- [Team Norms](project_kickoff/team_norms.md)
+- [Meeting Etiquette](project_kickoff/meeting-etiquette.md)
+- [Securing Your Workspace](project_kickoff/securing_your_workspace.md)
 
 ## Vision & Roadmap
 
-- [Customer Development](/vision_and_roadmap/customer_development_playbook.md)
-- [Product Speclet](/vision_and_roadmap/speclet.md) + [Template](/vision_and_roadmap/speclet_template.md)
-- [Visualizing the future of product initiatives as a cross-functional team](/vision_and_roadmap/visualizing-product-initiatives.md) + [Template](https://www.figma.com/file/kEVk8PaoLbCqI8liK5Emgh/%5BInitiative-Name%5D-Visualization-Summary?type=whiteboard&node-id=0%3A1&t=Dq0dcV75It3GqXxP-1)
+- [Customer Development](vision_and_roadmap/customer_development_playbook.md)
+- [Product Speclet](vision_and_roadmap/speclet.md) + [Template](/vision_and_roadmap/speclet_template.md)
```

Comment:
Should the second link have a leading slash? From my understanding above, it should not.

Reply:
Nope!  I'll correct it (though it does no harm in *this file*)